### DISCUSSION TITLE
BUG: concurrency with "cancel-in-progress: false" still cancel the next jobs instead of waiting.

### DISCUSSION THREAD

**Discussion by ItamarShalev** [community/community#53506] [Actions]
### Select Topic Area

Bug

### Body

I have a job called Test and two more jobs.
`name: Code checks

on:
  pull_request:
    branches: [ main ]
  push:
    branches: [ main, dev ]

jobs:
  Test:
    runs-on: ubuntu-latest
    concurrency:
      cancel-in-progress: false
      group: ${{ github.workflow }}-Test`

I want that every commit will run parallel to the two additional jobs, but for Test flow, only one check will run, I'm using the settings above.

But still, it cancels the next running instead of putting them in a queue and waiting.

Help ?

**Comment by TSRBerry** [comment]
See this discussion here: https://github.com/orgs/community/discussions/41518

**Reply by ItamarShalev** [reply]
I saw the same issue, only one run in the queue, but I didn't find any solution, isn't any solution or attempt to fix it?

**Reply by TSRBerry** [reply]
This is not a bug in the eyes of GH currently.

**Reply by ItamarShalev** [reply]
Is there a maintainer to check it? 
I saw on LinkedIn many people talk about it, and can't work with GitHub actions without it and using Jenkins instead.

**Comment by JamesKyburz** [comment]
Still a bug
